contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -213,6 +213,13 @@ func (t *T) AcceptOffer(offer *mesos.Offer) bool {
 	if offer == nil {
 		return false
 	}
+
+	// possibly check NodeName
```

Comment:
nit: could we improve this comment? something along the lines of:
if the user has specified a target host make sure it's aligned with this offer